packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -142,6 +145,46 @@ export const frLambdaTyped = (
   };
 };
 
+//This will box if not boxed. We could do a form that doesn't do this, but it would be messier.
+//I could see it as cleaner to use a new Class or interface like {type: "Boxed", value: T, args: BoxedArgs} | {type: "Unboxed", value: T}, but doesn't seem worth it yet.
```

Comment:
A different thing we could do is just use the type:
```{ value: T; args: BoxedArgs:undefined }```.

This is less semantic (it doesn't make it clear that BoxedArgs are undefined when that item is not boxed), but maybe that's fine - it's simple at least.